packages/firestore/package.json

```diff
@@ -11,7 +11,7 @@
     "lint:fix": "tslint --fix -p tsconfig.json -c tslint.json 'src/**/*.ts' 'test/**/*.ts'",
     "lint:eslint": "eslint -c .eslintrc.json '**/*.ts' --ignore-path '../../.gitignore'",    
     "prettier": "prettier --write 'src/**/*.js' 'test/**/*.js' 'src/**/*.ts' 'test/**/*.ts'",
-    "test": "run-s lint test:all",
+    "test": "yarn test:emulator",
```

Comment:
I think we still want to run the browser tests, so we should replace `test:node` with `test:emulator` in the "test:all" instead.

Reply:
How does changing `test:node` to `test:emulator` in `test:all` make browser tests run? My understanding is that `test:emulator` runs `test:node` against the emulator rather than prod.

Reply:
How does changing test:node to test:emulator in test:all make browser tests run? My understanding is that test:emulator runs test:node against the emulator rather than prod.

Reply:
Travis runs `yarn test` which in turn runs `yarn test:all`, so both node and browser tests will run.